articles/openshift/oshift-how-restrict-access-to-openshift-routes-by-ip.md

```diff
@@ -0,0 +1,57 @@
+---
+title: UKCloud Knowledge Centre article about whitelisting public IP's on your openshift routes | UKCloud Ltd
+description: Restrict access to openshift routes by public IP[s]
+services: openshift
+author: Mudasar Hussain
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: restrict access to openshift routes by public IP[s]
+toc_fullpath: How To/restricting-access-to-openshift-routes-by-ip.md
+toc_mdlink: oshift-how-restrict-access-to-openshift-routes-by-ip.md
+---
+
+# How to restrict access to a route in openshift by IP[s] address
+
+## Overview
+
+This knowledge centre article explains how openshift customers can restrict access to the routes they create to their deployed services by IP[s] addresses.
+
+### Intended audience
+
+Openshift developers who are looking to secure exposed routes of application by IP[s]
+
+### Bulleted lists
+
+  - Create and expose a route in the usual manner. Now you must add a annotation to the route along with the IP[s] that you would like to whitelist to be able to access the route. 
```

Comment:
I would change this to a normal paragraph (not a bullet) and rephrase to:

After creating and exposing the route in the usual way, you must add an annotation to the route along with the IP address(es) that you want to add to the whitelist, enabling them to access the route: